docs/rollout/focus.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Focus Users
+layout: rollout
+---
+
+Release Tilt first to an initial set of Focus Users so you can give them a great experience.
+
+## Focus is Easier to Support than Spread Wide
+
+My first thought when I configure a tool is to tell everyone on my team. This **Spread Wide** approach can go great, but it's risky.
```

Comment:
This is the only time you use the first person in this article and the inconsistency is off-putting. Try: "It can be tempting to introduce Tilt to everyone on your team at once. This approach carries several risks: "